backend/otel/otel.go

```diff
@@ -199,6 +200,12 @@ func (o *Handler) HandleTrace(w http.ResponseWriter, r *http.Request) {
 							continue
 						}
 
+						if fields.exceptionMessage == "" {
+							lg(ctx, fields).
```

Comment:
yes rn I'm just trying to see what these events are - since this branch only happens if the event is an exception, I want to make sure this isn't an exception we should be logging (using something other than `exception.message`)